core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -455,6 +455,10 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
   def createManagedGroup(groupRef: ManagedGroupRef):  Future[PerRequestMessage] = {
     for {
       _ <- samDAO.createGroup(WorkbenchGroupName(groupRef.membersGroupName.value), userInfo)
+      allUsersEmail <- dataSource.inTransaction { dataAccess =>
+        dataAccess.rawlsGroupQuery.loadEmails(Seq(allUsersGroupRef))
```

Comment:
I don't think you need to change anything, but the email address for the All_Users group is probably never going to change, or change so rarely that we can perhaps treat it as a constant?  So, should we just never make this query?  Just put the All_Users group details into a constant or at least a `lazy val` so it only needs to get loaded once?

Reply:
it doesn't change within an environment but does change from env to env. Loading it once would probably be ok but people don't create groups that often and it probably adds little overhead.